articles/matlab-image-database/index.md

```diff
@@ -0,0 +1,120 @@
+### How to create face database using matlab.
+### Introduction.
+Face databases are imagery data that are used for testing face processing algorithms. In the content of biometrics, databases are collected and used to evaluate the performance of the face recognition biometrics system.
+There are so many image databases available on the internet but if you want to make a database of your faces then you can use this program. The framework of this program can be used to create the database for other many objects.
+In this case, we will use the WEBCAM to make the number of required faces. Face database can be used to implement projects such as face recognition.
+
+### Prerequisites.
+1. Installed webcam.
+2. Proper understanding of the Ipcam and webcam.
+
+#### How to install the webcam.
+- In the home section click on the add-on then select the hardware support packages.
+- Scroll down and click on the ‘show all’ to find the package.
+- Select the imaging/cameras.
+- Click on the Matlab support package for USB packages.
+
+### Matlab code for creation of the face database.
```

Comment:
Before we proceed further, you could add an objective of what this Matlab code is related about? What is being demonstrated?

Reply:
Are you going to build a face recognition algorithm to detect faces? If so, is it just detection of face from the existing database? Or counting the faces? Or what operation is being performed exactly?